content/doc/book/using/best-practices.adoc

```diff
@@ -0,0 +1,143 @@
+This section describes best practices to provide executives, business managers, software developers, and architects with a better sense of what Jenkins can contribute throughout the project lifecycle.
+
+== Use organization folders
+
+Organization Folders offer a convenient way to allow Jenkins to automatically create and delete jobs as repositories and branches are created and deleted.
+Particularly, if you use GitHub organizations, Bitbucket teams, GitLab groups, or Gitea organizations, then Jenkins can automatically detect the creation of a new repository and create a Multibranch Pipeline project for it.
+
+See the link:/doc/book/pipeline-as-code/[organization folders] documentation for more details.
+
+.GitHub organization folders
+video::LbXKUKQ24T8[youtube,width=800,height=420]
+
+.GitLab group folders
+video::it6TOeQ6EHg[youtube,width=800,height=420]
+
+.Bitbucket project folders
+video::85b6fiVolfk[youtube,width=800,height=420]
+
+.Gitea organization folders
+video::NO3sZWRxgQM[youtube,width=800,height=420]
+
+== Use multibranch Pipelines
+
+If something prevents you from using organization folders, then use multibranch Pipelines.
+
+Insert more wisdom here ...
+
+See the link:/doc/book/pipeline-as-code/[multibranch Pipelines] documentation for more details.
+
+== Use Pipeline
+
+Insert Pipeline wisdom here
+
+See the link:/doc/book/pipeline/[Pipeline] documentation for more details.
+
+video::IOUm1lw7F58[youtube,width=800,height=420]
+
+== Use reports effectively
+
+Charts and graphs show trends and patterns that can highlight project status and progress.
+Automated test results for unit tests, integration tests, and end to end tests can highlight brittleness or instability.
+Coverage reports can illustrate areas that automated tests are not executing.
+Compiler warning messages can be a first indication of a problem.
+Static analysis tools can report risky code or code that may have security risks.
+Performance test results can idenitfy delays or problem areas.
+
+The plugin:warnings-ng[warnings next generation] plugin provides convenient access to many reports, including:
+
+* compiler warnnings and errors (like gcc, clang, javac, golang, etc.)
+* static analysis warnings and errors (like spotbugs, checkstyle, pmd, lint, cpd, Simian, etc.)
+* code coverage reports
+
+video::tj3xYFA6Q2o[youtube,width=800,height=420]
+
+== Show failures to the right people
+
+Tune the notifications from failing and unstable jobs so that the notifications reach the right people without distracting others.
+Many Jenkins users prefer to only be notified when a failure is likely their fault.
+They correctly assume that if they did not cause the failure, they are not the best person to investigate the failure.
+
+Refine your notification system so that new test failures first notify the most recent committers, since they are the most likely cause of the issue.
+
+.Sending Slack notifications
+video::EDVZli8GdUM[youtube,width=800,height=420]
```

Comment:
```suggestion
Configure the notifications from failing and unstable jobs so the notifications reach the right people without distracting others.
Many Jenkins users prefer to only be notified when a failure is likely their fault.
They correctly assume that if they did not cause the failure, they are not the best person to investigate the failure.

Refine your notification system so that new test failures first notify the most recent committers, since they are the most likely cause of the issue.

.Sending Slack notifications
video::EDVZli8GdUM[youtube,width=800,height=420]
```
suggestion to update some of the syntax